dapp/client/App.elm

```diff
@@ -96,18 +116,49 @@ view model =
                 , div [ class "value" ] [ text <| showAddress model.address ]
                 ]
             ]
+        , viewWallet model
         ]
 
 
 showAddress : Maybe Address -> String
-showAddress x =
-    withDefault "No Address Available" x
+showAddress address =
+    address
+        |> withDefault "No Address Available"
+
+
+viewWallet : Model -> Html Message
+viewWallet model =
+    let
```

Comment:
Let me know if you have questions about this part. It's a view function to handle displaying the wallet section of the page.